include/msgpack/object.h

```diff
@@ -97,6 +97,8 @@ typedef struct msgpack_object_kv {
     msgpack_object val;
 } msgpack_object_kv;
 
+int msgpack_object_init(msgpack_object* d, void* data, size_t size, int type);
```

Comment:
```suggestion
MSGPACK_DLLEXPORT
bool msgpack_object_init(msgpack_object* d, int type, void* data, size_t size);
```

1. `MSGPACK_DLLEXPORT` should be added because other similar functions have it. (For Windows DLL support)
2. Please update parameter order. 
3. Please use `bool` return type for success/fail result.
4. Please apply coding rule. `void *data` -> `void* data`. (Some exisiting code violates it anyway)